storage/perfschema/table_replication_applier_status.h

```diff
@@ -60,6 +60,12 @@ struct st_row_applier_status {
   uint remaining_delay;
   bool remaining_delay_is_set;
   ulong count_transactions_retries;
```

Comment:
There is a pre-existing inconsistency in the code here. :warning:

- `count_transactions_retries` is defined as `ulong` in the header.
- `COUNT_TRANSACTIONS_RETRIES` is defined as [`BIGINT unsigned`](https://mariadb.com/kb/en/bigint/) in the SQL definition:
   https://github.com/MariaDB/server/blob/182ff21ace34ea4f00fb5b66689b172323d91f99/storage/perfschema/table_replication_applier_status.cc#L66
- The value of the `COUNT_TRANSACTIONS_RETRIES` field in the query output row is set using `set_field_ulonglong`: 
   https://github.com/MariaDB/server/blob/182ff21ace34ea4f00fb5b66689b172323d91f99/storage/perfschema/table_replication_applier_status.cc#L231-L233

Reply:
can you correct this mistake for 10.6 and put new functionality into 11.4 branch

Reply:
> can you correct this mistake for 10.6

It's an inconsistency. How do I know which definition is correct or intended? 32 bit, or 64 bits?

Note that *MySQL* has the exact same inconsistency: https://github.com/search?q=repo%3Amysql%2Fmysql-server%20COUNT_TRANSACTIONS_RETRIES&type=code

Reply:
@grooverdan, I've rebased _this one_ onto 11.4.

Reply:
1. I prepended the commit df490e8 which fixes the preexisting data type inconsistency, and
2. I've also spun that off as https://github.com/MariaDB/server/pull/2905, based on 10.6